changelog.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## 0.12.1~rc1
+
+### Web Application
+
+* Add "Back to submission page" link to NoScript docs (#4208)
+
+### Operations
+
+* Ensured `gcc` and WiFi related packages are not installed on Xenial on upgrade (#4163)
```

Comment:
`gcc` will still installed per: https://github.com/freedomofpress/securedrop/issues/4163#issuecomment-468681362 , apologies should have changed the issue name.

Reply:
Ok, so this is still only issue 4163 and the change would be ```s/`gcc` and//``` basically yeah?

Reply:
yup!